### ISSUE TITLE
when traing, the loss becomes nan. why?

### ISSUE THREAD

**Issue by zhouwy19** [sgrvinod/a-PyTorch-Tutorial-to-Object-Detection#38]
![image](https://user-images.githubusercontent.com/43036573/73132481-def22e00-4056-11ea-8d21-a369b0e7b929.png)

**Comment by abhisingh977** [comment]
Hey,
Are you using the same dataset?
If not then this is happening because the lr is high for your dataset. Reduce the lr.
As soon as the loss reaches around thousands it starts showing nan or inf.

Go to train.py and reduce the learning rate(lr).

**Comment by tripathiarpan20** [comment]
Upon some tracing I found out that MultiboxLoss returns that nan value, maybe this is the same issue as #54
